Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -114,11 +119,8 @@ public void setTaskStatusDown(Task task) {
      *            object
      */
     public void setTasksStatusUp(Process process) throws DataException, IOException {
-        List<Task> tasks = new CopyOnWriteArrayList<>(process.getTasks());
-
-        for (Task task : tasks) {
-            setTaskStatusUp(task);
-        }
+        Task currentTask = ServiceManager.getProcessService().getCurrentTask(process);
```

Comment:
`getCurrentTask` should not be used anymore as there might be more than one current task per process.